pycds/weather_anomaly.py

```diff
@@ -193,6 +203,70 @@ class MonthlyAverageOfDailyMaxTemperature(Base, MaterializedViewMixin):
     __selectable__ = monthly_average_of_daily_temperature_extremum_selectable('max')
     __primary_key__ = 'history_id vars_id obs_month'.split()
 
+
 class MonthlyAverageOfDailyMinTemperature(Base, MaterializedViewMixin):
     __selectable__ = monthly_average_of_daily_temperature_extremum_selectable('min')
     __primary_key__ = 'history_id vars_id obs_month'.split()
+
+
+# TODO: Factor out common subquery for non-discarded obs_raw_id's (as a view)
```

Comment:
It might be a good idea to do this now rather than leaving it as a TODO. When will we ever come back to this?